monitor/election.go

```diff
@@ -0,0 +1,33 @@
+package monitor
+
+import (
+	"context"
+
+	"github.com/celo-org/kliento/contracts"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+func electionLogProcessor(ctx context.Context, logger log.Logger, electionAddress common.Address, election *contracts.Election, eventLog *types.Log) {
+	logger = logger.New("contract", "Election")
+	if eventLog.Address == electionAddress {
+		eventName, eventRaw, ok, err := election.TryParseLog(*eventLog)
+		if err != nil {
+			logger.Warn("Ignoring event: Error parsing election event", "err", err, "eventId", eventLog.Topics[0].Hex())
```

Comment:
probably good to create a metric on this.... since it's sign of an outdated abi

Reply:
Good idea https://github.com/celo-org/eksportisto/issues/2